apis/civo/network/v1alpha1/types.go

```diff
@@ -0,0 +1,60 @@
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CivoNetworkConfig specs for the CivoNetwork
+type CivoNetworkConfig struct {
+	Label  string `json:"label"`
+	Region string `json:"region"`
+}
+
+// CivoNetworkSpec holds the networkConfig
+type CivoNetworkSpec struct {
+	xpv1.ResourceSpec `json:",inline"`
+	NetworkConfig     CivoNetworkConfig `json:"networkConfig"`
```

Comment:
Can we use the fields here directly i.e?
```suggestion
	Label  string `json:"label"`
	Region string `json:"region"`
```